### DISCUSSION TITLE
Starting page size of articles

### DISCUSSION THREAD

**Discussion by MichaelMGK** [templaza/astroid-framework#206] [Q&A] [answered]
Hello, 
I am searching for a solution who I can resize articles on my starting page to an indivdual size. Currently I use 2 columns to show highlighted articles. The window of two articles is everytime the same high (of the larger article). How I can individualize it? Somethink like Tiles from different high. 

Any idea if this is possible?

BR
Michael

![astroid](https://user-images.githubusercontent.com/112613958/193407597-5ecaaab9-5932-440f-8153-b4e3088912dc.jpg)

On our old website we use an additional plugin for it, but I want to reduce additional software (and it is not available for J4)
![old_website](https://user-images.githubusercontent.com/112613958/193407667-abaa38e2-120e-4a70-9b6d-586c36ff1282.jpg)

**Comment by Chacapamac** [comment]
Do you have a live link?

**Reply by Chacapamac** [reply]
Maybe you can find your answer here —> https://forum.joomla.org/viewtopic.php?t=973106

**Reply by MichaelMGK** [reply]
Yes, here it is: https://www.mariae-himmelfahrt-mh.de/

**Comment by sonvnn** [comment, marked as answer]
@MichaelMGK 
You can use Custom CSS. Add this code below:
```
.article-wraper {
    height: auto;
}
```

**Reply by Chacapamac** [reply]
So Simple Thanks Sonny, I put that in my library.
Good Day!

**Reply by MichaelMGK** [reply]
Great. Thanks!

**Comment by Chacapamac** [comment]
Well, I just look again and the .article-wrapper css does not give the “Masonry” effect that you need as it still leave big white space between articles, or I’m wrong?

I find this that explain how to make a “Masonry” effect with bootstrap 4 using the “card” element
From Bootstrap — > https://getbootstrap.com/docs/4.0/components/card/#card-columns
From Stackoverflow —> https://stackoverflow.com/questions/48468140/bootstrap-4-masonry-layout-utilizing-flexbox-grid

My suggestion will be to create an override of the layout you are using

**Reply by MichaelMGK** [reply]
you are right. there are now white spaces, but it is better than having large frames. My CSS and programmer expertice is very low, so building a override for the design is a inpossible challenge for me.

**Comment by Chacapamac** [comment]
**@sonvnn I mark this as an enhancement,** 
it will be great to have an option to permit a Masonry look in the blog view in Astroid
